.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'tag/version'
+        required: true
+        default: '1.0.0'
+      action_tag:
+        description: 'create tag ("no" to skip)'
+        required: true
+        default: 'yes'
+jobs:
+  release_roku:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: main
+
+    - name: Check version
+      run: |
+        set -eo pipefail
+        echo Target version: ${{ github.event.inputs.tag }}
+        make check-version VERSION=${{ github.event.inputs.tag }}
+    - name: Build Artifacts
+      if: ${{ github.event.inputs.action_tag == 'yes' }}
+      run: |
+        make archive
+    - name: Create GH Release
+      id: create_release
+      uses: release-drafter/release-drafter@v5
+      if: ${{ github.event.inputs.action_tag == 'yes' }}
+      with:
+        name: v${{ github.event.inputs.tag }}
+        tag: ${{ github.event.inputs.tag }}
+        version: ${{ github.event.inputs.tag }}
+        publish: true
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Upload - AEP Roku SDK
```

Comment:
During our discussion, we agreed to add the sample channel to the release page. Can we add it to this PR?

Reply:
From our discussion we will address this in a separate PR